### PR TITLE
small tweak

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala [hunk 1]
```diff
@@ -248,7 +248,7 @@ class LDACommander @Inject() (
       scored.filter { _._2 < MAX_KL_DIST }.sortBy(_._2).take(topK).map { _._1 }
     }
 
-    val userFeats = db.readOnlyReplica { implicit s => uriTopicRepo.getUserRecentURIFeatures(userId, wordRep.version, min_num_words = 50, limit = 50) }
+    val userFeats = db.readOnlyReplica { implicit s => uriTopicRepo.getUserRecentURIFeatures(userId, wordRep.version, min_num_words = 50, limit = 100) }
```
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: Can we make it 200? Or do you think that will be too slow?

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: OK. will try. Note: with this kind of retrieval, we almost could have a recent interest score based on KL divergence.

**Comment by yingjieMiao** [comment]
@stkem 

updated.

File: kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: True. However, this call is much less frequent than the interest scoring.

**Comment by stkem** [comment]
:+1:

**Pull request merged**